safe-global/safe-wallet-web
# feat: add owner flow

## What it solves

Migrating the add owner flow to the new transaction flow framework.

## How this PR fixes it

Each step of the add owner flow has been ported, as well as removing the old one.

Note: the remove owner flow uses elements of the add owner one and certain aspects of it have been commented out in order to keep this only add owner focused.

## How to test it

Add an owner via the settings and observe each step working correctly as well as a successful transaction.

## Screenshots

![add owner flow](https://github.com/safe-global/safe-wallet-web/assets/20442784/a2d1e5ad-997d-474c-8e0d-7eb632bd7351)

## Checklist
